src/components/dialog/examples/dialog-closing-actions.tsx

```diff
@@ -2,7 +2,36 @@ import { Component, h, State } from '@stencil/core';
 
 /**
  * Custom closing actions
+ *
+ * Action buttons in dialogs can be used to add a clear visual indication for
+ * the sighted users to realize that the dialog can be closed by pressing
+ * a button as well.
+ * This may sometimes be considered an unnecessary usage of action buttons for
+ * sighted users. Because majority of them users know that clicking or tapping
+ * outside the dialog closes it.
+ *
+ * Such buttons usually are labeled as `OK`, `Dismiss` or `Close`.
```

Comment:
Backticks formats the text as code. That _looks_ good in this case, but I don't think it's semantically good… I think quote marks, cursive text, and/or bold text would serve the purpose better.

```suggestion
 * Such buttons are usually labeled ***OK***, ***Dismiss*** or ***Close***.
```